src/Microsoft.Azure.SignalRService/IServiceHubContext.cs

```diff
@@ -0,0 +1,13 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using Microsoft.AspNetCore.SignalR;
+
+namespace Microsoft.Azure.SignalRService
+{
+    public interface IServiceHubContext: IDisposable, IHubContext<Hub>
+    {
+        IUserGroupManager UserGroups { get; }
```

Comment:
>UserGroups [](start = 26, length = 10)

naming `UserGroupManager`

Reply:
I think `UserGroups` is better, since there is a property `IGroupManager Groups { get; }` In `IHubContext`. To make naming consistent, I name it `UserGroups`.